android/test_app/app/build.gradle

```diff
@@ -25,17 +31,37 @@ android {
         }
         externalNativeBuild {
             cmake {
+                abiFilters ABI_FILTERS.split(",")
                 arguments "-DANDROID_STL=c++_shared"
             }
         }
         buildConfigField("String", "MODULE_ASSET_NAME", "\"mobilenet2q.pt\"")
         buildConfigField("String", "LOGCAT_TAG", "@string/app_name")
         buildConfigField("long[]", "INPUT_TENSOR_SHAPE", "new long[]{1, 3, 224, 224}")
+        buildConfigField("boolean", "NATIVE_BUILD", 'false')
         addManifestPlaceholders([APP_NAME: "@string/app_name", MAIN_ACTIVITY: "org.pytorch.testapp.MainActivity"])
     }
+    signingConfigs {
+        config {
+            storeFile file("testapp.keystore")
```

Comment:
curious: what's the common practice to handle keystore for demo/test app? I found this but not sure whether it's worth the trouble for test app: https://stackoverflow.com/questions/44911547/where-to-store-android-keystore-file-for-cirlceci-build

Reply:
Yes, we can do it more securely for CircleCI. That keystore is completely dummy, there is no 'PyTorch' and 'Facebook' inside it. 
I think if somebody will start signing his app with this keystore there should not be any problems.